docs/src/site/twiki/FalconDocumentation.twiki

```diff
@@ -615,10 +617,35 @@ Failure to follow any of the above rules would result in a process submission fa
 present in the system for the specified time period, the process can be submitted and scheduled, but all instances
 created would result in a WAITING state unless data is actually provided in the cluster.
 
+---++ Updating cluster entity definition
+User would want to update the cluster entity in following scenarios
+   * when hadoop clusters are updated from unsecure to secure
+   * when hadoop clusters move from non high-availability to high-availability etc
 
+In these scenarios, user would want to change the cluster entity to reflect updated interface endpoints or properties.
+Updating cluster would require cascading update to feed and process entity jobs in the workflow engine. So Falcon restricts
+Cluster update to following
+   * Cluster update can only be performed by superuser and when Falcon Server is in safemode.
+   * The underlying namenode or workflow engine referenced by interface URI is the same. It is only the URI that has changed to reflect secure/HA environments.
+
+Cluster entity should be updated by superuser using following CLI command.
+<verbatim>
+bash$ falcon entity -type cluster -name primaryCluster -update -file ~/primary-updated.xml
```

Comment:
Feel it is better to refer to CLI tutorial page, so this document provides a high-lever introduction and users can refer to CLI/API tutorials for the command they need. Same for the other CLI command you listed below.